curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-priority-queue-class.md

```diff
@@ -12,11 +12,15 @@ In this challenge you will be creating a Priority Queue. A Priority Queue is a s
 
 For instance, let’s imagine we have a priority queue with three items:
 
-`[['kitten', 2], ['dog', 2], ['rabbit', 2]]`
+```js
+[['kitten', 2], ['dog', 2], ['rabbit', 2]]
+```
 
 Here the second value (an integer) represents item priority. If we enqueue `['human', 1]` with a priority of `1` (assuming lower priorities are given precedence) it would then be the first item to be dequeued. The collection would look like this:
 
-`[['human', 1], ['kitten', 2], ['dog', 2], ['rabbit', 2]]`.
+```js
+[['human', 1], ['kitten', 2], ['dog', 2], ['rabbit', 2]]
+````
```

Comment:
Are the 4 backticks intentional?

Reply:
No, that was not intentional.  I will push up another commit to remove.